src/components/VTextField/VTextField.js

```diff
@@ -148,9 +175,8 @@ export default {
         const height = this.$refs.input
           ? this.$refs.input.scrollHeight
           : 0
-        const minHeight = this.rows * 24
-        const inputHeight = height < minHeight ? minHeight : height
-        this.inputHeight = inputHeight + (this.textarea ? 4 : 0)
+        const minHeight = this._rows * this._rowHeight
```

Comment:
just do `const minHeight = parseInt(this.rows) * parseInt(this.rowHeight)`

Reply:
Wilco (and the above too)